src/components/select/select.scss

```diff
@@ -58,13 +58,14 @@
 
         &:focus {
             outline: none;
-            .mdc-select__dropdown-icon {
-                box-shadow: var(--button-shadow-hovered);
-            }
+
             .mdc-floating-label {
                 color: var(--mdc-theme-primary);
             }
         }
+        &:focus-visible {
```

Comment:
WOOT!

NICE!

Not supported in Safari yet, but that seems like something we can live without, right? And for Mobile Safari it doesn't seem like the lack of support should be much of an issue, since it's probably not _very_ common for people to connect keyboards and tab through fields on their iPads or iPhones.

Reply:
Tnx :) 

I think we can use this until the new IE (Safari) realizes that they should also support it 😄